Generation/src/components/ConstPileUp.cpp

```diff
@@ -0,0 +1,40 @@
+
+#include "ConstPileUp.h"
+
+#include "GaudiKernel/DeclareFactoryEntries.h"
+
+DECLARE_TOOL_FACTORY( ConstPileUp )
+
+ConstPileUp::ConstPileUp( const std::string& type,
+                          const std::string& name,
+                          const IInterface* parent )
+    : GaudiTool ( type, name , parent ) {
```

Comment:
Please reformat according to coding style:
https://github.com/HEP-FCC/FCCSW/blob/master/doc/CppCodingStyleGuidelines.md#indentation